entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/MetisDereferenceService.java

```diff
@@ -85,22 +86,58 @@ public void afterPropertiesSet() throws MalformedURLException {
 
 
 
-    String fetchMetisResponse(String externalId) {
+    String fetchMetisResponse(String externalId) throws Exception {
```

Comment:
I would not recommend catching Exception and throwing Exception again. Can't we have a custom Exception class like EntityRunTimeException for such cases which gives an Internal Server Error and that extends the EuropeanaApiException as well?